SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/ClientServerRPCService.cpp

```diff
@@ -224,7 +224,6 @@ void ClientServerRPCService::OnEndpointAuthorityGained(const Worker_EntityId Ent
 		break;
 	}
 	default:
-		checkNoEntry();
 		break;
```

Comment:
Can I suggest a comment and maybe a reference to the UNR to explain why this default is ok / needed as a fix?